model/qti/Service.php

```diff
@@ -143,6 +146,14 @@ public function getXmlByRdfItem(core_kernel_classes_Resource $item, $language =
      */
     public function saveDataItemToRdfItem(Item $qtiItem, core_kernel_classes_Resource $rdfItem)
     {
+        if ($this->getFeatureFlagChecker()->isEnabled('FEATURE_FLAG_UNIQUE_NUMERIC_QTI_IDENTIFIER')) {
```

Comment:
Not captured by QA, but should we also add it for tests?

Reply:
As I see, we do not have tests for this method at all 😓 
But I will try to cover at least this method